default.nix

```diff
@@ -83,7 +84,8 @@ let
 
     # Make sure buildroot Python loads dynamic modules from the right place
     export LD_LIBRARY_PATH=$PWD/buildroot/output/host/usr/lib:/lib:/usr/lib
-    export PATH=$PWD/scripts/wrappers/bin:$PWD/buildroot/output/host/bin:$PATH
+    export PATH=$PWD/scripts/wrappers/bin:$PWD/buildroot/host_output/host/bin:$PWD/buildroot/output/host/bin:$PATH
```

Comment:
let's make sure this covers DEVC-1196 or communicate whether or not these changes are related